### PR TITLE
Ribocode

### PR DESCRIPTION
**Pull request by xzt41**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by xzt41** [comment]
Finally， I succeeded!

File: recipes/ribocode/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  version: 1.2.6
+
+source:
+  fn: RiboCode-1.2.6.tar.gz # [py2k]
+  url: https://pypi.python.org/packages/c6/4b/902412b7c9683bcb9096e9359d17c6c0538b084a0c3167603ed69278aeb6/RiboCode-1.2.6.tar.gz # [py2k]
+  md5: e05ef843da351c4588e02bad3d5c3d65 # [py2k]
+
+requirements:
+  build:
+    - python # [py27]
```
Comment (bgruening) [inline recipes/ribocode/meta.yaml]: can you please remove the pinning here (# [py27]) ... you skip py3k build already.

File: recipes/ribocode/build.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```
Comment (bgruening) [inline recipes/ribocode/build.sh]: you can remove this file entirely as you have this command already in your meta.yaml file

Comment (xzt41) [inline recipes/ribocode/build.sh]: I have removed this file.

File: recipes/ribocode/meta.yaml (hunk 1 above)
Comment (xzt41) [inline recipes/ribocode/meta.yaml]: I have modified this file, thank you !

**Comment by bgruening** [comment]
Thanks @xzt41!

**Pull request merged**